resources/renameFileExtensions.js

```diff
@@ -0,0 +1,30 @@
+const copy = require('copy');
+const path = require('path');
+const fs = require('fs');
+const os = require('os');
+
+const tmpDir = os.tmpdir();
+const [, , src, dest, destExtension] = process.argv;
+if (!src || !dest || !destExtension) {
+  console.error(
+    `\nMissing arguments.\n\nUsage:\nnode renameFileExtensions.js './dist/**/*.js' './dest-dir' .new.extension.js`,
+  );
+  process.exit(1);
+}
+
+const tempDirectory = fs.mkdtempSync(`${tmpDir}${path.sep}`);
+
+copy(src, tempDirectory, (error, files) => {
+  if (error) {
+    throw error;
+  }
+  files.forEach(file => {
+    if (file.dest) {
```

Comment:
When would there not be `file.dest`? Should it throw in that case?

Reply:
good question. this script is rewritten in step 3 significantly anyways, otherwise it doesnt work on netlify. the `fs.mkTempSync()` fails miserably on netlify but works fine on travis. Should i just backport those improvements to this PR then?

Reply:
That, or just throw an error in the `else` block here.

Reply:
I think I will do both. because this PR would fail on netlify right now.